_posts/2019-09-30-10.md

```diff
@@ -55,11 +54,43 @@ categories:
 * 13 的 Apple [#ArcadeReview](https://twitter.com/search?q=%23ArcadeReview%20%40ethanhuang13) 連載中
 * iPad mini 超適合玩 Apple Arcade 的，又輕又快又平💸
 
-## 歡樂的時光總是過得特別快
+## 本集播出後的後續討論與內容更正
+
+本集內容播出後，有推友向我們回饋有些內容觀念混淆也提出了正確和清楚的說法，非常感謝這些推友們，我們非常樂見這樣的~~抓漏~~討論！這些討論和更正整理如下：
+
+### Swift 的 closure 與 ObjectiveC 的 block
```

Comment:
Objective-C

Reply:
Fixed in 76fdf087a457b34dcc0d757011949fd3d832c02e